rest/taskana-rest-spring-example-boot/pom.xml

```diff
@@ -50,6 +50,36 @@
                     <version>${project.version}</version>
                 </dependency>
             </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <version>${version.maven.resources}</version>
+                        <executions>
+                            <execution>
+                                <id>copy-history-rest-documentation-to-static-folder</id>
+                                <phase>prepare-package</phase>
```

Comment:
Because this here is only necessary when we package the artifact while taskana-simplehistory-rest-spring needs the parsed javadoc from taskana-rest-spring in order to execute the tests.